packages/issuer-api/src/pods/certification-request/handlers/create-certification-request.handler.ts

```diff
@@ -12,6 +12,12 @@ import { CertificationRequestDTO } from '../certification-request.dto';
 import { CertificationRequest } from '../certification-request.entity';
 import { CreateCertificationRequestCommand } from '../commands/create-certification-request.command';
 
+const eth = /^(0x)[0-9a-f]{40}$/i;
```

Comment:
This implementation assumes that `a-f` are always lower-case, which is not true for checksummed addresses. Sometimes they are also `A-F`.